nextcord/channel.py

```diff
@@ -689,6 +689,10 @@ async def create_thread(
 
         .. versionadded:: 2.0
 
+        .. note::
+
+            This will raise :exc:`HTTPException` if the guild features isn't contain ``PRIVATE_THREADS`` and you are trying to create a private thread.
```

Comment:
```suggestion
            This will raise :exc:`HTTPException` if the guild features does not include ``PRIVATE_THREADS`` and you are trying to create a private thread.
```